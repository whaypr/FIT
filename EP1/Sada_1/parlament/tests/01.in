4
10 10
1 2 3 4 5 6 7 8 9 10
1 2 3 4 5 6 7 8 9 10
1 2 3 4 5 6 7 8 9 10
1 2 3 4 5 6 7 8 9 10
1 2 3 4 5 6 7 8 9 10
1 2 3 4 5 6 7 8 9 10
1 2 3 4 5 6 7 8 9 10
1 2 3 4 5 6 7 8 9 10
1 2 3 4 5 6 7 8 9 10
1 2 3 4 5 6 7 8 9 10
9
1 1 2 9
10 10 10 10
1 1 1 10
10 1 10 10
1 1 1 1
2 2 2 2
1 1 10 10
9 9 10 10
2 2 9 9
1 1
1
1
1 1 1 1
2 3
1 2 3
1 2 3
4
1 1 2 3
1 1 2 2
1 2 1 2
1 2 2 3
4 2
1 2
3 4
5 6
7 8
5
1 1 2 2
3 2 4 2
2 1 2 2
1 1 4 1
1 2 4 2