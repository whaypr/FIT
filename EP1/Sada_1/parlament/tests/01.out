Absolutni hodnota pohodlnosti je 90 bodu.
Absolutni hodnota pohodlnosti je 10 bodu.
Absolutni hodnota pohodlnosti je 55 bodu.
Absolutni hodnota pohodlnosti je 55 bodu.
Absolutni hodnota pohodlnosti je 1 bodu.
Absolutni hodnota pohodlnosti je 2 bodu.
Absolutni hodnota pohodlnosti je 550 bodu.
Absolutni hodnota pohodlnosti je 38 bodu.
Absolutni hodnota pohodlnosti je 352 bodu.

Absolutni hodnota pohodlnosti je 1 bodu.

Absolutni hodnota pohodlnosti je 12 bodu.
Absolutni hodnota pohodlnosti je 6 bodu.
Absolutni hodnota pohodlnosti je 2 bodu.
Absolutni hodnota pohodlnosti je 10 bodu.

Absolutni hodnota pohodlnosti je 10 bodu.
Absolutni hodnota pohodlnosti je 14 bodu.
Absolutni hodnota pohodlnosti je 7 bodu.
Absolutni hodnota pohodlnosti je 16 bodu.
Absolutni hodnota pohodlnosti je 20 bodu.
